### ISSUE TITLE
Rule proposal: Switch braces

### ISSUE THREAD

**Issue by Richienb** [sindresorhus/eslint-plugin-unicorn#1702]
### Description

See https://github.com/xojs/xo/issues/650 for additional context.

### Fail

```js
switch (value) {
		case 'unicorn':
			break;
}
```

### Pass

```js
switch (value) {
		case 'unicorn': {
			break;
		}
}
```

**Comment by fisker** [comment]
Let's implement this rule with `always` and `avoid` option?

**Comment by jonahsnider** [comment]
The Rome JavaScript toolchain has a rule that curly braces are only required when there is more than one expression in the `case` statement.
